packages/neuron-wallet/tests/services/tx/transaction-generator.test.ts

```diff
@@ -1196,7 +1197,7 @@ describe('TransactionGenerator', () => {
           const targetOutput: Output = Output.fromObject({
             capacity: toShannon('61'),
             lock: aliceAnyoneCanPayLockScript,
-            type: assetAccountInfo.generateSudtScript('0xuuid'),
+            type: assetAccountInfo.generateSudtScript('0x1234'),
```

Comment:
Does this change matter in this refactor?

Reply:
Yes, the `0xuuid` is not a valid hex string, `lumos` serilization will check it and throw `invalid hex string` error.

Reply:
Got it.